assets/scripts/lobby/chat.js

```diff
@@ -454,11 +454,11 @@ function addToRoomChat(data) {
 }
 
 function isPlayerMuted(username) {
-  if (mutedPlayers.indexOf(username) !== -1) {
-    return true;
+  if (!username) {
+    return false;
   }
 
-  return false;
+  return mutedPlayers.indexOf(username.toLowerCase()) !== -1;
```

Comment:
Feels like this line alone should be enough

Reply:
Can you also double check that when players are muted, their names are lowercased as well?

Reply:
The mutedplayers list for players is all in lowercase